pkg/publish/default.go

```diff
@@ -119,6 +119,52 @@ func pushResult(tag name.Tag, br build.Result, opt []remote.Option) error {
 	}
 }
 
+func (d *defalt) MultiPublish(in map[string]build.Result) (map[string]name.Reference, error) {
+	ro := []remote.Option{remote.WithAuth(d.auth), remote.WithTransport(d.t)}
+	no := []name.Option{}
+	if d.insecure {
+		no = append(no, name.Insecure)
+	}
+
+	out := map[string]name.Reference{}
+	m := map[name.Reference]remote.Taggable{}
+	for ip, br := range in {
+		// https://github.com/google/go-containerregistry/issues/212
+		sanitized := strings.TrimPrefix(strings.ToLower(ip), build.StrictScheme)
+		for _, tagName := range d.tags {
+			tag, err := name.NewTag(fmt.Sprintf("%s/%s:%s", d.base, d.namer(sanitized), tagName), no...)
+			if err != nil {
+				return nil, err
+			}
+			if err != nil {
+				return nil, err
+			}
+			m[tag] = br
+		}
+
+		h, err := br.Digest()
+		if err != nil {
+			return nil, err
+		}
+		ref := fmt.Sprintf("%s/%s@%s", d.base, d.namer(sanitized), h)
```

Comment:
I think there's a merge conflict with my change here.
```suggestion
		ref := fmt.Sprintf("%s@%s", d.namer(d.base, sanitized), h)
```